assets/index.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/extensions */
+import Player from './player.js';
+import Table from './board.js';
+
```

Comment:
clear code!
kindly move the DOM functions to another file.
![image](https://user-images.githubusercontent.com/31897434/76318591-39c9a700-62de-11ea-8586-973cc1f2aad9.png)

Reply:
Done!